_posts/2018-10-30-conducting-good-retrospectives.md

```diff
@@ -0,0 +1,110 @@
+
+Leaving the meeting with a list of action items ensures that the tasks you spent an hour finding, **actually get done**.
+
+## Case Study
+
+For the last year, my team has had a weekly retro, alternating the format each week. We operate in two week sprints, so in the middle of the sprint we typically concentrate on interpersonal issues or other general frustrations and at the end of the sprint we focus on project-specific issues.
+
+However, at the beginning of any retrospective, we solicit the opportunity to suggest an alternative format.
+
+#### Mid-Sprint Retrospective
+
+During the Mid-Sprint Retrospective, we engage in the [Lean Coffee](http://leancoffee.org) activity (unless someone suggests an alternate activity!). The chosen facilitator will explain the chosen activity and pass out the required supplies. 
+
+The basics of the Lean Coffee activity are:
+
+1. Generate discussion topics
+    - Limit to 5 minutes
+1. Vote on topics
+1. Discuss topics
+    - Limit to 5 minutes, then vote on continuing with a 2 minute extension, or else move on to next topic.
+    - Facilitator will take notes and track action items.
+
+Any topics that aren't discussed are carried over to the next meeting, unless the author states otherwise.
+
+#### Sprint Retrospective
+
+During the Sprint Retrospective we use the [Four Quadrant](http://www.funretrospectives.com/lessons-learned-quadrants-planning-vs-success/) activity. As with the Mid-Sprint retro, the chosen facilitator will explain the activity and pass out the required supplies.
+
+The basics of the Four Quadrant activity are:
+
+1. Enumerate events that happened during the last two weeks on sticky notes.
+    - Limit to 5 minutes
+1. The facilitator will draw a horizontal axis on the white board, with the left side indicating _bad_ and the right side _good_.
+1. Each person will give a brief description of their stickies and place them on the axis where they think they belong.
+1. The facilitator then draws a vertical axis, with the top indicating the event was _in the team's control_ and the bottom that it was _not in the team's control_.
+1. The facilitator quickly pulls each sticky to the appropriate place on the vertical axis. This is usually fairly obvious, but the rest of the team will signal where they think it should go in case of disagreement.
+
+At this point, the board should resemble something like this:
+
+![Diagram of a completed Four Quadrants diagram]({{ 'https://res.cloudinary.com/mhanberg/image/upload/v1541686229/four-quadrants.png' }})
+
+Starting in the top right corner and moving counter clockwise, the team discusses the events in an attempt to learn. If your team is prone to drawing out discussions, consider using a Lean Coffee style discussion timer.
+
+**Keep doing** - good and in your control: give yourselves a quick pat on the back to celebrate good work.
```

Comment:
Potentially also discuss how to enable the team to continue these things (if, say, a stick was about an experiment).